src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingResponseMapper.java

```diff
@@ -30,14 +31,23 @@
 import uk.gov.hmcts.reform.hmc.model.UnavailabilityRanges;
 import uk.gov.hmcts.reform.hmc.model.hmi.HearingResponse;
 import uk.gov.hmcts.reform.hmc.model.hmi.RequestDetails;
+import uk.gov.hmcts.reform.hmc.repository.HearingPartyRepository;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 @Component
 public class GetHearingResponseMapper extends GetHearingResponseCommonCode {
 
+    private HearingPartyRepository hearingPartyRepository;
```

Comment:
This feels all kinds of wrong to have a repository within a mapper ?

Reply:
Removed, Fixed by @danlysiak in https://github.com/hmcts/hmc-cft-hearing-service/pull/153/commits/50b833069211b0402b67717fae973ed563286907